postcss.config.js

```diff
@@ -1,6 +1,8 @@
 module.exports = {
   plugins: [
     require('postcss-import'),
+    require('tailwindcss'),
+    //require('autoprefixer'),
```

Comment:
Everything says we should have this uncommented but I'm not sure why yet. I'm leaving it commented for now so if something doesn't work later I can remember to try uncommenting this, but I'll remove it if we don't use it once we're going.

Reply:
From what I understand, tailwind doesn't need autoprefixer to work, but it does create css rules that may require vendor prefixes in some browsers. [This is the best I can find right now for proof of this](https://github.com/tailwindlabs/tailwindcss/issues/291)

Reply:
@neenjaw Thanks 🙂  Turns out that `postcss-preset-env` already has support for auto-prefixing so this isn't required at all.